### DISCUSSION TITLE
Horizontal Scrolling Text

### DISCUSSION THREAD

**Discussion by alvinlys** [swiftbar/SwiftBar#329] [General]
Hi, new to here. May i know does anyone knows how to create horizontal scrolling text by using the Plugin API? thanks

**Comment by melonamin** [comment]
Hi @alvin30595, currently it can’t be done. SwiftBar supports clipping long text, but no option for horizontal scrolling. 

What kind of plugin are you trying to build?

**Reply by alvinlys** [reply]
Oh, was thinking to build some spotify/music player plugin where the song & artist can be displayed in fixed width with scrolling.
